### PR TITLE
new icon: netbeans (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #930: Icon request: NetBeans
### About the icon

Netbeans, also called Apache Netbeans is a Java IDE made by Apache

### Links and sources

Website: https://netbeans.apache.org/
SVG Icon: https://netbeans.apache.org/images/apache-netbeans.svg

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by canaleal**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #930**

## Link to prove your SVG is correct and up-to-date.

- [Branding](https://www.apache.org/logos/#netbeans)

The color `#1e6abf` was selected for the font SVGs as it's the most common color in the original-wordmark logo.

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -6632,6 +6632,29 @@
             }
         ]
     },
+    {
+        "name": "neatbeans",
```
Comment (weh) [inline devicon.json:6636]: there is a typo
```suggestion
        "name": "netbeans",
```

**Review by weh — CHANGES_REQUESTED** [review]
Just a typo, other than that looks good.

File: devicon.json (hunk 1 above)
Comment (Snailedlt) [reply devicon.json:6636]: I for one, think neatbeans is a pretty neat name!

**Review by weh — APPROVED** [review]
LGTM :+1:

**Review by weh — APPROVED** [review]

**Comment by canaleal** [comment]
The SVGs have no artifacts. 

image

image

**Pull request merged**